public/pages/PreviewDetector/containers/AdjustModel.tsx

```diff
@@ -90,6 +90,12 @@ export function AdjustModel(props: AdjustModelProps) {
       );
     }
   };
+  const flyoutStyleTop = props.isSticky ? '111px' : `${224 - props.topOffset}px`;
+  const flyoutStyle = {
+    top: flyoutStyleTop,
+    borderTop: '0px',
+    height: 'calc(100% - ' + flyoutStyleTop +')',
```

Comment:
You can use string litterate similar to line 93
```
height: `calc(100%-${flyoutstypeTop})`
```

Reply:
thanks. Have used this way.